modules/installation/pages/2-pre-installation/1-usb-creation.adoc

```diff
@@ -0,0 +1,186 @@
+
+== Downloading
+
+The Fawkes tarball needs to be downloaded.
+
+NOTE: For the time being, the artifacts are downloaded individually and from a private mirror.
+In the near future these will be available in a single artifact from a public mirror.
+
+=== Prerequisites
+
+The following prerequisites are necessary for completing this guide.
+
+* `curl`
+
+NOTE: Windows users can download link:https://curl.se/windows/[`curl` for Windows].
+
+=== Downloading the Fawkes tarball
+
+Fawkes is only available from a private mirror at this time.
+
+Using the private mirror requires credentials to https://artifactory.algol60.net. For information on obtaining these, see link:https://app.tango.us/app/workflow/Downloading-from-Private-Repos-in-Artifactory-eb4f579cdf9a4ea8988e402a18008b7b[this guide].
+
+. Set Credentials
+* Username and token (see link:https://app.tango.us/app/workflow/Downloading-from-Private-Repos-in-Artifactory-eb4f579cdf9a4ea8988e402a18008b7b[this guide] for assistance).
++
+[source,bash]
+----
+ARTIFACTORY_USER=
+----
++
+[source,bash]
+----
+read -s ARTIFACTORY_TOKEN
+----
+. Set a desired release.
+* A specific release.
++
+[source,bash]
+----
+RELEASE=0.0.1-alpha.3
+----
+* The latest release from stable or unstable
++
+NOTE: At this time, Fawkes is still in a pre-release state and the stable stream is empty.
++
+[source,bash]
+----
+STREAM=unstable
+RELEASE="$(curl -u "${ARTIFACTORY_USER}":"${ARTIFACTORY_TOKEN}" "https://artifactory.algol60.net/artifactoy/api/search/latestVersion?g=${STREAM}&a=fawkes")"
+----
+. Download the tarball
++
+NOTE: To use a proxy, add `--proxy http://example.com:443` to each `curl` call.
++
+[source,bash]
+----
+if [[ "$RELEASE" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+    STREAM=stable
+else
+    STREAM=unstable
+fi
+curl -C - -o "fawkes-${RELEASE}.tar.gz" \
+"https://$ARTIFACTORY_USER:$ARTIFACTORY_TOKEN@artifactory.algol60.net/artifactory/fawkes-releases/${STREAM}/fawkes/${RELEASE}/fawkes-${RELEASE}.tar.gz"
+----
+
+== Creating the USB
+
+This segment will walk the user through creating the USB with the downloaded tarball.
+
+=== Linux
+
+The Linux method will enable persistence as well as data storage for bringing artifacts into the booted LiveCD.
+
+==== Prerequisites
+
+Prerequisites for creating the USB from a Linux box.
+
+* `blkid`
+* `dd`
+* `mktemp`
+* `parted`
+* `sgdisk`
+* `sha256sum`
+
+==== Steps
+
+The following steps will create a USB from a Linux OS.
+
+. Extract all images and RPMs from the tarball.
++
+[source,bash]
+----
+tar --wildcards -xzvf ./fawkes-${RELEASE}.tar.gz \
+    "*/images/" \
+    "*/rpm/"
+----
+. Validate the image's SHA sums
++
+[source,bash]
+----
+for image_dir in fawkes-${RELEASE}/images/*; do
+    (
+    cd "$image_dir"
+    sha256sum -c ./"$(basename "$image_dir")"*sha256.txt
+    )
+done
+----
+All the images should return with `OK`, if any fail, please stop and open a bug (use the bug opening tab on the bottom right of this screen).
```

Comment:
All of the images should return with `OK`. If any fail, please stop and open a bug by using the bug opening tab on the bottom right of this screen.